src/Chapter3.hs

```diff
@@ -560,22 +647,26 @@ introducing extra newtypes.
 🕯 HINT: if you complete this task properly, you don't need to change the
     implementation of the "hitPlayer" function at all!
 -}
+newtype Damage = Damage { damageValue :: Int}
+newtype Defense = Defense { defenseValue :: Int}
+newtype Health = Health { healthValue :: Int }
+
 data Player = Player
-    { playerHealth    :: Int
+    { playerHealth    :: Health
     , playerArmor     :: Int
     , playerAttack    :: Int
     , playerDexterity :: Int
     , playerStrength  :: Int
     }
```

Comment:
You can go even further and introduce the newtype for each of the fields here to gain maximum benefit from the type system :slightly_smiling_face:

You can read more about the `newtype` pattern in our blog:

* https://kowainik.github.io/posts/haskell-mini-patterns#newtype